docs/error-reporting/platform-integrations/source-map.md

```diff
@@ -56,64 +66,101 @@ uglifyjs main.js -c -m --source-map -o main.min.js
 </TabItem>
 </Tabs>
 
-**Step 2: Configure @backtrace-labs/javascript-cli**
+---
+### Step 2: Install `backtrace-js`
+
+Install the `backtrace-js` command line tool and update your build scripts to run it. `backtrace-js` can be run from the command line, but it is most efficient to use a configuration file which  we will create in the next step.
 
 1. Install `@backtrace-labs/javascript-cli` as a dev dependency:
 
    ```bash
-   npm install --save-dev @backtrace-labs/javascript-cli
+   npm install -g --save-dev @backtrace-labs/javascript-cli
    ```
 
-2. Add the following scripts to your `package.json` file:
+2. Add the following script to `package.json` to process and upload source maps:
 
    ```json
    "scripts": {
-     "backtrace:process": "backtrace-js process OUTPUT_DIRECTORY",
-     "backtrace:upload": "backtrace-js upload OUTPUT_DIRECTORY UPLOAD_OPTIONS"
+     // highlight-next-line
+     "backtrace:sourcemap": "backtrace-js run",
    }
    ```
 
-You can also use `npx` and skip adding the dependency:
+3. Update the build step in `package.json` to process and upload source maps with every build:
 
-```json
-"scripts": {
-  "backtrace:process": "npx --package @backtrace-labs/javascript-cli backtrace-js process OUTPUT_DIRECTORY",
-  "backtrace:upload": "npx --package @backtrace-labs/javascript-cli backtrace-js upload OUTPUT_DIRECTORY UPLOAD_OPTIONS"
-}
-```
-
-Replace `OUTPUT_DIRECTORY` with the path to the directory where your transpiled scripts are stored.
-
-Replace `UPLOAD_OPTIONS` with `--url <your upload URL>`, obtained from [the symbol submission URL](#acquire-a-symbol-submission-token-and-url).
+   ```json
+   "scripts": {
+      // highlight-next-line
+     "build": "<current build commands> && npm run backtrace:sourcemap"
+   }
+   ```
 
-### Configuration File
+---
+### Step 3: Create a `.backtracejsrc` configuration file
 
-Instead of providing options in script argument lines, you can configure them in the `.backtracejsrc` configuration file:
+Create a `.backtracejs` file in the root of your project.
 
-```jsonc
+```json
 {
-  "path": "OUTPUT_DIRECTORY",
-  "upload": {
-    "url": "your upload URL"
-  }
+    // highlight-next-line
+    "path": "<build output>",
+    "run": {
+        "process": true,
+        "add-sources": true,
+        "upload": true
+    },
+    "upload": {
+        // highlight-next-line
+        "url": "<symbol submission url>",
+        "include-sources": true
```

Comment:
maybe it's worth to also explain each parameter? for example: include-sources will add your source code information to source map report. With this option enabled you can see your source code in Backtrace.

Reply:
Added a detailed note explaining how source files are uploaded with source maps, and provided an alt path for suppressing that in favor of integrating a source repo.